src/main/java/net/bootsfaces/menu/ComponentListBean.java

```diff
@@ -87,9 +87,6 @@ public String getDisplayName(String c) {
 	}
 	
 	public String addSlashIfNecessary(String path) {
-		if (path.length() <= 1) {
-			return path;
-		}
```

Comment:
a function with the name addSlashIfNecessary and an exit by length only make no sense